spec/spec_helper.rb

```diff
@@ -0,0 +1,83 @@
+require_relative '../book'
+require_relative '../classroom'
+require_relative '../decorate'
+require_relative '../person_class'
+require_relative '../rental'
+require_relative '../student_class'
+require_relative '../teacher_class'
+
+describe Book do
+  before :each do
+    @book = Book.new('All girls are the same', 'Tobby')
+  end
+
+  describe '#addnew' do
+    it 'takes the two parameter' do
+      expect(@book).to be_an_instance_of Book
+    end
+  end
+
+  describe 'check value of title and author' do
+    it 'value of title' do
+      expect(@book.title).to eq('All girls are the same')
+    end
+
+    it 'value of author' do
+      expect(@book.author).to eq('Tobby')
+    end
+  end
+end
+
+describe 'decorator' do
+  before :each do
+    @person = Person.new(22, 'oluwatobiloba')
+  end
+  it 'change name to capitalize' do
+    @capitalize = CapitalizeDecorator.new(@person)
+    expect(@capitalize.correct_name).to eq('Oluwatobiloba')
+  end
+  it 'change name to capitalize' do
```

Comment:
- [ ] It would be better if you change here the text to be test the trimmer of the word if it has more than 10 charcters instead of capitalize since it is testing the trim method 🔥✅💯🚀